Cargo.toml

```
@@ -18,11 +18,12 @@ keywords = ["data-structure", "no_std"]
 categories = ["data-structures", "no-std"]
 
 [dependencies]
-serde = { version = "1.0", optional = true, features = ["derive"] }
+serde = { version = "1.0", optional = true, default-features = false, features = ["alloc", "derive"] }
```

```suggestion
serde = { version = "1.0.95", optional = true, default-features = false, features = ["alloc", "derive"] }
```
ref: https://github.com/serde-rs/serde/commit/50ebbd63c6097fe927c92e577defba9c49d6793d

---

changed in b3d0924